.gitignore

```diff
@@ -170,3 +170,4 @@ cython_debug/
 
 .ijwb
 *.db
+*.db-journal
```

Comment:
This one was just something I noticed pop up that we don't want in general.

Reply:
Out of curiosity, is this something from sqlite and when is that generated? (I never noticed that!)
Thanks for catching it btw, I think it makes sense not to commit it

Reply:
I'm not sure, but I guess so as that's all I've been testing with 🤔 ... odd.

Reply:
Just did a quick check online to satisfy myself, looks like it usually creates the file during transactions, but cleans up when done. So maybe it got created as something was killed off half way through (allowing it to sort things out when it came back up).